addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -229,7 +229,7 @@ def _notify_typing(self, is_typing):
             :param is_typing: (boolean) tells whether the members are typing or not
         """
         for member in self:
-            member.channel_id._bus_send_store(member, extra_fields={"isTyping": is_typing})
+            member.channel_id._bus_send_store(member, extra_fields={"isTyping": is_typing, "isTypingDt": fields.Datetime.now()})
```

Comment:
Note: field/variable names referenced in python should be snake case (even if not "real" field)